### ISSUE TITLE
Change genres color

### ISSUE THREAD

**Issue by debs-obrien** [debs-obrien/playwright-movies-app#44]
Change the color of genres to a purple color ensuring color is ok for accessibility. The selected purple should have sufficient contrast ratio with its background to meet WCAG accessibility standards.

**Comment by debs-obrien** [comment]
Here is a screenshot of the app running in development mode:

![App Screenshot](http://localhost:3000/?category=Popular&page=1)

**Comment by debs-obrien** [comment]
Here is a PDF screenshot of the app running in development mode:

[App Screenshot](https://github.com/debs-obrien/playwright-movies-app/files/1/page-2025-03-21T13:43:11.502Z.pdf)

**Comment by debs-obrien** [comment]
Here is a PDF screenshot of the app running in development mode:

[App Screenshot](https://github.com/debs-obrien/playwright-movies-app/blob/main/page-2025-03-21T13:46:00.546Z.pdf)
